env/work/srw_python/uti_math.py

```diff
@@ -612,12 +612,23 @@ def trf_rotation(_V, _ang, _P):
     return [M, V]
 
 #****************************************************************************
-def fwhm(x, y): #MR27092016
-    """The function searches x-values (roots) where y=0 based on linear interpolation, and calculates FWHM"""
+def fwhm(x, y, shift=0.5, return_as_dict=False):  # MR21032017
```

Comment:
Rename to something more general, e.g. statistical moments (peak value, total flux, ...).